x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/datafeed/DatafeedConfig.java

```diff
@@ -519,6 +570,17 @@ public void setScrollSize(int scrollSize) {
             this.scrollSize = scrollSize;
         }
 
+        public void setDelayedDataCheckWindow(TimeValue delayedDataCheckWindow) {
+            if (delayedDataCheckWindow != null) {
```

Comment:
Why should that matter? When doing the check, I round down the start/end values so that they themselves are multiples of the bucket_span.

Reply:
in other similar cases we validate the window is a multiple of the bucket span to avoid the implicit handling of this. I don't think it's a problem here, I was asking to understand the behaviour.